HybridLocalTemplate/install.js

```diff
@@ -10,6 +10,17 @@ for (var sdkDependency in packageJson.sdkDependencies) {
     var repoUrlWithBranch = packageJson.sdkDependencies[sdkDependency];
     var parts = repoUrlWithBranch.split('#'), repoUrl = parts[0], branch = parts.length > 1 ? parts[1] : 'master';
     var targetDir = path.join('mobile_sdk', sdkDependency);
+
+    // Exclude tagged SalesforceMobileSDK-Android releases so it's only a source dependency pre-release.
+    if (sdkDependency == 'SalesforceMobileSDK-Android' && branch.match(/v\d+\.\d+\.\d+/)) {
+       if (fs.existsSync(targetDir)) {
```

Comment:
@wmathurin - Take a look at this for https://github.com/forcedotcom/SalesforceMobileSDK-Templates/pull/377#discussion_r1353763217

Reply:
Looks good. 

In terms of messaging, what do you think about doing the following:
```javascript
console.log(`Using ${skdDependency} at ${branch}`);
if (sdkDependency == 'SalesforceMobileSDK-Android' && branch.match(/v\d+\.\d+\.\d+/)) {
    if (fs.existsSync(targetDir)) {
        console.log(`Warning: Published artifacts will not be used since sources are already in ${targetDir}. If desired, remove this directory to return to the release artifacts.`);    
    }
   continue; // Getting SDK from Maven Central
}
```

Reply:
@JohnsonEricAtSalesforce Maybe :arrow_up: in the install.js ?